tmc-plugin/src/fi/helsinki/cs/tmc/actions/CheckForNewExercisesOrUpdates.java

```diff
@@ -76,118 +78,80 @@ public CheckForNewExercisesOrUpdates(boolean beQuiet, boolean backgroundCheck) {
 
     @Override
     public void actionPerformed(ActionEvent e) {
+        logger.info("action performed");
```

Comment:
Should we log the name of the action too, or does to logging framework already log the location of the call?